app/common/lib/historyUtil.js

```diff
@@ -144,6 +145,19 @@ const getKey = (siteDetail) => {
   return null
 }
 
+const isHistorySite = (site) => {
+  if (!site) {
+    return false
+  }
+  const order = site.get('order')
+  const location = site.get('location')
+  const tags = site.get('tags')
```

Comment:
Is this check added, because we still have `tags` property on other devices, because `tags` are removed from `browser-laptop` repo?